src/api/firebase.js

```diff
@@ -4,6 +4,7 @@ import {
 	doc,
 	setDoc,
 	getCountFromServer,
+	updateDoc,
```

Comment:
Awesome job digging through the documentation and thank you for the notes left on your pull request very helpful in understanding the Firebase methods!